src/App.js

```diff
@@ -1,13 +1,67 @@
 import React from 'react';
 
 import './App.scss';
+import { Goods } from './Goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  displayAll = () => {
+    getAll().then((goodsFromServer) => {
+      this.setState({ goods: goodsFromServer });
+    });
+  }
+
+  display5First = () => {
+    get5First().then((goodsFromServer) => {
+      this.setState({ goods: goodsFromServer });
+    });
+  }
+
+  displayRed = () => {
+    getRed().then((goodsFromServer) => {
+      this.setState({ goods: goodsFromServer });
+    });
+  }
```

Comment:
these functions could be ONE function, which receives callback and set state with its value ;)